ckan/templates/page.html

```diff
@@ -45,7 +45,7 @@
             </div>
           {% endblock %}
 
-          <div class="row wrapper{% block wrapper_class %}{% endblock %}{% if self.secondary()|trim == '' or c.action=='resource_read' %} no-nav{% endif %}">
```

Comment:
This change is not entirely necessary for this PR but I think it is time to remove it.

I have created also a PR for this: https://github.com/ckan/ckan/pull/7165/files so I can keep working on it instead of here.